test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -74,3 +74,17 @@ def test_path_from_cli_depend_on_cwd(base_arguments, monkeypatch):  # Issue 572
 
     assert 'test/test' not in options1.exclude_paths[0]
     assert 'test/test' in options2.exclude_paths[0]
+
+
+def test_invalid_config_file(base_arguments):
+    """Ensures invalid config_file produces error code 2."""
+    with pytest.raises(SystemExit, match="^2$"):
+        cli.get_config(base_arguments +
+                       ["-c", "test/fixtures/ansible-config-invalid.yml"])
+
+
+def test_missing_config_file(base_arguments):
+    """Ensures missing config_file produces error code 2."""
+    with pytest.raises(SystemExit, match="^2$"):
+        cli.get_config(base_arguments +
+                       ["-c", "/dev/null/ansible-config-missing.yml"])
```

Comment:
We should parametrize this, you're copy-pasting too many lines. Also, I'd check for the message in stderr using the `capsys` fixture.

Reply:
For 3 tests maybe it would have made sense, but for only two is going to add complexity. Still, I will do it.

Not testing stderr/stdout output was on purpose because the only contract I want to offer is the result code. The content of the error message and where it is sent may change in time, especially as we are likely to do some refactoring and adopt python logging.